File: Armen/mario.c
#include <stdio.h>
#include <cs50.h>
int main(void){
    int height=get_int("input height: ");
    for (int i=1; i<=height; i++){
        for (int n=i; n<height; n++){
            printf(" ");
        }
        for (int n=1; n<=i; n++){
            printf("#");
        }
        printf(" | ");
        for (int n=1; n<=i; n++){
            printf("#");
        }
        
        printf("\n");
    }
}

File: Garnik/CS50/Week6/Mario/mario.py
from cs50 import get_int

height = 0

# prompt the user input if the height is not possitive number
while height < 1 or height > 8:
    height = get_int("Input the height: ")

# print spaces and dashes
for i in range(1, height+1):
    print(" " * (height-i) + "#" * i + "  " + "#" * i)
